nncf/structures.py

```diff
@@ -30,11 +30,13 @@ class QuantizationPrecisionInitArgs(NNCFExtraConfigStruct):
     :param data_loader: 'data_loader' - provides an iterable over the given dataset, instance of descendant
                 of 'torch.utils.data.DataLoader' class. Must return both inputs and targets to calculate loss
                 and gradients.
+    :param device: Device to perform initialization at. Either 'cpu' or 'cuda' (default).
     """
 
-    def __init__(self, criterion: _Loss, data_loader: DataLoader):
+    def __init__(self, criterion: _Loss, data_loader: DataLoader, device: str = 'cuda'):
```

Comment:
I think, better to use torch.device type

Reply:
Probably `str` is what the users are more used to. Will also help generalize for TF.